certified-connectors/iObeya/apiDefinition.swagger.json

```diff
@@ -0,0 +1,598 @@
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "inner",
+              "middle",
+              "outer"
+            ],
+            "default": "outer",
+            "description": "Wedge Ring (inner, middle, outer)",
+            "x-ms-summary": "Wedge Ring (inner, middle, outer)"
+          }
+        ]
+      }
+    },
+    "/cards/activity": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "self": {
+                  "type": "string",
+                  "description": "self"
+                },
+                "kind": {
+                  "type": "string",
+                  "description": "kind"
+                },
+                "totalCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Total Count Elements"
+                },
+                "previous": {
+                  "type": "string",
+                  "description": "Previous page Url"
+                },
+                "next": {
+                  "type": "string",
+                  "description": "Next page Url"
+                },
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "self": {
+                        "type": "string",
+                        "description": "Url of the element"
+                      },
+                      "kind": {
+                        "type": "string",
+                        "description": "kind"
+                      },
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "Title"
+                      },
+                      "startDate": {
+                        "type": "string",
+                        "description": "Start date of the activity"
+                      },
+                      "endDate": {
+                        "type": "string",
+                        "description": "End date of the activity"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "List Cards Activity",
+        "description": "List Cards Activity between two dates",
+        "operationId": "ListCardsActivity",
+        "parameters": [
+          {
+            "name": "boardId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Board where retrieve activity card",
```

Comment:
@smiquerolle please make sure everything is grammatically correct and is free of typographical errors.